src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: $transition-speed;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here, but it is not defined within this file. Ensure that this variable is defined in a global SCSS file or imported correctly to avoid compilation errors.